packages/legend-art/src/utils/TextEditorUtils.ts

```diff
@@ -118,6 +120,17 @@ export const setErrorMarkers = (
   ]);
 };
 
+export const setAllWarningMarkers = (
+  editorModel: monacoEditorAPI.ITextModel,
+  allWarnings: monacoEditorAPI.IMarkerData[],
+): void => {
+  monacoEditorAPI.setModelMarkers(editorModel, DUMMY_MARKER_OWNER, allWarnings);
```

Comment:
--> `setWarningMarkers()`. not sure why we parse in `IMarkerData`, we should follow `setErrorMarkers`, we enforce the severity and only let people pass in a list of coordinates (i.e. `TextEditorRange`)

Let's create 2 types:

```ts
export type TextEditorPosition = {
  lineNumber: number;
  column: number;
};

export type TextEditorRange = {
  startLineNumber: number;
  startColumn: number;
  endLineNumber: number;
  endColumn: number;
};

```